docs/installing.md

```diff
@@ -0,0 +1,80 @@
+# Running a local Haveno test network
+
+These are the steps needed to setup a local Haveno test network.
+
+## 1. Download dependencies
+
+1. Install git, git-lfs and Java. On ubuntu: `sudo apt install git git-lfs openjdk-11-jdk`
+2. Download the Monero binaries. To make things easier, we provide two custom Monero binaries: `monerod` and `monero-wallet-rpc`. You can directly [download the software (Linux and Mac)](https://github.com/haveno-dex/monero/releases/tag/testing) or build the binaries by yourself, by cloning and building this repository: https://github.com/haveno-dex/monero
```

Comment:
> https://github.com/haveno-dex/monero doesn't have the right commit history.

It does. It's the branch you gave me with the commit history we need. It's compared against master but it's based on a branch (which differs from master)

> How about forking from monero-project for a direct upstream link?

The only difference that would make is that it would show *forked from monero..* on top, but i would have to force push that branch anyway. This gave me more flexibility. I can fork again from monero and then force push your branch if preferred, but as said that would only change the link to the repo.

Reply:
> I can fork again from monero and then force push your branch if preferred

Yeah having monero-project directly upstream is preferred.

Reply:
> We might want to preserve the branches so it's a direct mirror + added capability for Haveno.

I avoided that because it's more straightforward for a contributor to clone the main branch instead of specifying one, but sure we can do it.